plans/response-parser-plan.md

```
@@ -0,0 +1,188 @@
+| text               | `text/plain`, `text/html`, `text/csv`          | `.text()` | `string`     |
+| text (XML-based)   | `image/svg+xml`, `application/xml`, `text/xml` | `.text()` | `string`     |
+| binary (images)    | `image/*` (except svg+xml)                     | `.blob()` | `Blob`       |
+| binary (media)     | `audio/*`, `video/*`                           | `.blob()` | `Blob`       |
+| binary (generic)   | `application/octet-stream`                     | `.blob()` | `Blob`       |
+| binary (documents) | `application/pdf`, `application/zip`, etc.     | `.blob()` | `Blob`       |
+| multipart          | `multipart/*`                                  | `.blob()` | `Blob`       |
+| none               | no `content-type` header or empty body         | `return;` | `undefined`  |
+
+## Parser Selection Logic
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)